tests/specs/functional/lemmas-spec.k

```diff
@@ -504,7 +504,6 @@ module LEMMAS-SPEC
  // bitwise
  // -------
 
-    claim [bit-n005-0]: <k> runLemma ( notMaxUInt5   &Int X ) => doneLemma( 0 ) ... </k> requires #rangeUInt (   5 , X )
```

Comment:
The four `005` lemmas are not needed, as the illustrative examples are about manipulation in multiples of 8.